Docs/source/index.rst

```diff
@@ -96,7 +96,6 @@ Data Analysis
    dataanalysis/visit
    dataanalysis/visualpic
    dataanalysis/picviewer
-   dataanalysis/backtransformed_diags
```

Comment:
Did we remove the file already?
Shall we (in a follow-up) add a new one? cc @RevathiJambunathan ?